package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "galileo",
-  "version": "1.23.0",
+  "version": "1.24.0",
```

Comment:
Im not sure if it matters but you may want to also run `npm run build` so that you get package-lock.json updated as well (it contains the version)